tokenExchange/src/app/tokenGen.js

```diff
@@ -51,6 +51,13 @@ function getTokenComponent(decodedToken, keyId) {
     payload.organization = organization;
   }
 
+  const source = {};
+  if (decodedToken.source) {
```

Comment:
Qui non viene mandata l'informazione sul retrivalId o in generale sul token/id inviato dalla sorgente

Reply:
se il token che sta gestendo la lambda non ha le informazioni riguardanti la sorgente non le inserisce nel token di sessione

Reply:
si corretto ma quando la source c'è il retrival id non viene rimandato indietro. ricordi quel discorso di ritornarlo per permettere a noi front-end di recuperarlo quando si fa reload della pagina?

Reply:
si mi era saltato il requisito... sistemo